oem/ibm/libpldmresponder/utils.cpp

```diff
@@ -436,6 +436,39 @@ void hostChapDataIntf(
         "/xyz/openbmc_project/pldm", dbusToFilehandlerObj);
 }
 
+uint32_t getLinkResetInstanceNumber(std::string& path)
+{
+    uint32_t id = pldm::dbus::CustomDBus::getCustomDBus().getBusId(path);
+    return id;
+}
+
+void findSlotObjects(const std::string& boardObjPath,
+                     std::vector<std::string>& slotObjects)
+{
+    static constexpr auto MAPPER_BUSNAME = "xyz.openbmc_project.ObjectMapper";
```

Comment:
can we make the change to reduce the dbus string constants like this - https://gerrit.openbmc.org/c/openbmc/pldm/+/69194 ?